src/main/java/org/web3j/tx/HighThroughputTransactionManager.java

```diff
@@ -0,0 +1,127 @@
+package org.web3j.tx;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.request.RawTransaction;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import rx.Subscription;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * High Throughput transaction manager.
+ */
+
+public class HighThroughputTransactionManager extends RawTransactionManager {
```

Comment:
Hey @ferOnti - do you need any further input from me on creating an integration test for the HighThroughputTransactionManager. I see you responded with a confused emoticon.